app.rb

```diff
@@ -0,0 +1,172 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def start
+    puts 'School library App!'
+    menu
+  end
+
+  def menu
+    loop do
+      puts ' ' * 10
+      puts 'Enter a number from 1 to 7 to pick your option:'
+      puts '1 - List of all books available'
+      puts '2 - List of all people available'
+      puts '3 - Add a person'
+      puts '4 - Add a book'
+      puts '5 - Add a rental'
+      puts '6 - List all rentals for a person id'
+      puts '7 - Exit the library'
+      puts ''
+
+      choice = gets.chomp.to_i
+
+      case choice
+      when 1..6 then app.chosen(choice)
+      when 7 then puts 'Thank you for using this application, come back soon.'
+      end
+      break if choice == 7
+    end
+  end
+
+  def chosen(choice)
+    case choice
+    when 1
+      list_all_books
+    when 2
+      list_all_people
+    when 3
+      create_a_person
+    when 4
+      puts create_a_book
+    when 5
+      create_a_rental
+    when 6
+      list_all_rentals
+    end
+  end
+
```

Comment:
[REQUIRED] This file is doing too much. Kindly move the option selection to the main.rb file. This helps with the separation of concerns. Kindly look at this lesson for more guidance:
https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point